compiler/include/resolution.h

```diff
@@ -29,7 +29,9 @@ class CallInfo;
 
 extern SymbolMap paramMap;
 extern Vec<CallExpr*> callStack;
-extern Map<Type*,FnSymbol*> autoCopyMap; // type to chpl__autoCopy function
+bool hasAutoCopyForType(Type* type);
```

Comment:
Or we could just put autoCopyMap to be an exported symbol again.

The point of these changes was to simplify the common case for getting the auto copy function from the map. In the common case, if it's NULL, it should be an error. So getAutoCopyForType fails with an error if it's about to return NULL.

As a separate matter, adding these "accessor functions" enables future modifications to do with the auto-copy map.

I think it's clearer to use hasAutoCopyForType to get the uncommon case - where not having an autoCopy is tolerable - to work. (And as an aside, I think tolerating a NULL auto-copy-map entry is a workaround anyway).